<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelSeclet</class>
 <widget class="QWidget" name="ChannelSeclet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="ChannelMap" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>583</width>
     <height>324</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>583</width>
     <height>324</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="allSelect">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noSelect">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>全不选</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_keyboardinput">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>220</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_No">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>第  通道自检电阻:  
(欧姆)</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>641</x>
     <y>171</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="lb_No_2">
      <property name="text">
       <string>桥路:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_value1">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>641</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lb_No_3">
      <property name="text">
       <string>绝缘:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_value2">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="selswitch">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>绝缘通道</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
